packages/wallet/src/util.ts

```diff
@@ -0,0 +1,9 @@
+// 3k/f (where k is the security parameter in genesis, and f is the active slot co-efficient parameter
+// in genesis that determines the probability for amount of blocks created in an epoch.)
+
+import { Cardano } from '@cardano-sdk/core';
+
+export const calculateStabilityWindowSlotsCount = ({
+  securityParameter,
+  activeSlotsCoefficient
+}: Cardano.CompactGenesis) => (3 * securityParameter) / activeSlotsCoefficient;
```

Comment:
Please add the return type (number?)

Reply:
Applied in https://github.com/input-output-hk/cardano-js-sdk/pull/433/commits/4c407e6e7521733781853cc636c544f6613584ff